### PR TITLE
infocards made

### LINKED ISSUE #3: Info Cards
Has food cards

### PR DESCRIPTION
**Pull request by JustinLung**
closes #3

### CODE REVIEW

File: src/components/InfoCard/InfoCard.module.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  padding: 1rem 1.5rem 0;
+  font-size: 2.2rem;
+  font-weight: 600;
+}
+
+.info-card__description {
+  margin: 0;
+  padding: 1rem 1.5rem 1.5rem;
+  font-size: 1.125rem;
+  font-weight: lighter;
```
Comment (DaanKorver) [inline src/components/InfoCard/InfoCard.module.css]: Please increase font-weight according to design

File: src/components/InfoCard/InfoCard.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  infoCardTitle: string,
+  infoCardDescription: string
+}
+
+export function InfoCard({ infoCardImage, infoCardTitle, infoCardDescription }: Props) {
+  return (
+    <>
+      <article className={styles['info-card']}>
+        <div className={styles['info-card__image-wrapper']}>
+          <Image src={infoCardImage} alt="Food" width={500} height={500} className={styles['info-card__image-wrapper__image']} />
```
Comment (DaanKorver) [inline src/components/InfoCard/InfoCard.tsx:15]: Add gradient overlay just like in the design

File: src/components/OurDishes/OurDishes.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import styles from './OurDishes.module.css'
+import { InfoCard } from "../InfoCard/InfoCard"
+
+function InfoCardContainer() {
+  return (
+    <>
+      <section className={styles['our-dishes']}>
+        <h2 className={styles['our-dishes__title']}>Our Dishes</h2>
+        <div className={styles['our-dishes__info-card-container']}>
+          <InfoCard infoCardImage="/assets/images/food.png" infoCardTitle="Hotpot" infoCardDescription="With our hotpot,  we’ve managed to put a new spin on an old classic. Simplicity is the name of the game: we use fresh ingredients and offer a variety of high-quality food at an affordable price. Are you ready for visiting us with friends and family?" />
```
Comment (DaanKorver) [inline src/components/OurDishes/OurDishes.tsx]: Maybe add a temporary for loop to simulate a 3 card layout.

**Review by DaanKorver — CHANGES_REQUESTED** [review]

**Comment by JustinLung** [comment]
@DaanKorver fixed the ammends.

**Pull request closed without merging**